.circleci/config.yml

```diff
@@ -73,7 +87,6 @@ jobs:
           at: ~/repo/
       - run: pip install --upgrade pip
       - run: pip install -r lambda-layers/DependenciesLayer/requirements.txt
-      - run: make collectstatic
```

Comment:
I've tried to run `collectstatic` as `pipenv run python manage.py collectstatic` as well and both throws an error for `export PIPENV_VENV_IN_PROJECT=1` on line 67 although I'm not sure why.

Reply:
Based on comparing to the website repo CI build, I think what you've done here (moving to running `pipenv run collectstatic` after we call `make lambda-layers/DependenciesLayer/requirements.txt` in the `install_and_update_dependencies` job) looks right :+1:
Lets review if we try a staging deploy and have no stylesheets/js etc.